support/remote_health_monitoring/using-insights-operator.adoc

```diff
@@ -0,0 +1,29 @@
+:_content-type: ASSEMBLY
+[id="using-insights-operator"]
+= Using Insights Operator
+include::_attributes/common-attributes.adoc[]
+ifdef::openshift-dedicated[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+endif::[]
+:context: using-insights-operator
+
+toc::[]
+
+The Insights Operator periodically gathers anonymized configuration and component failure status and, by default, reports that data every two hours to Red Hat. This information enables Red Hat to assess configuration and deeper failure data than is reported through Telemetry. Users of {product-title} can display the report in the Insights Advisor service on link:https://console.redhat.com/openshift[Red Hat Hybrid Cloud Console].
+
+[role="_additional-resources"]
+.Additional resources
+
+* The Insights Operator is installed and enabled by default. If you need to opt out of remote health reporting, see xref:../../support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc#opting-out-remote-health-reporting[Opting out of remote health reporting].
+
+* See, xref:../../support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc#using-insights-to-identify-issues-with-your-cluster[Using Insights to identify issues with your cluster] for more information on using Insights Advisor to identify issues with your cluster.
+
+* See, link:https://github.com/openshift/insights-operator/blob/master/docs/gathered-data.md[] for a list of Insights Operator gather operations. 
+
+* See, link:https://github.com/openshift/insights-operator/blob/master/docs/anonymized-data.md[] for a list of anonymized data points in the Insights Operator archive.
+
+* Insights Operator is maintained from a public repository located at link:https://github.com/openshift/insights-operator[].
```

Comment:
"Avoid linking to GitHub. Instead, try to incorporate the required information directly into our documentation so that it goes through the same QE verification process as the rest of the content. Exceptions to this rule must be okayed by the relevant Product Experience team rep (for OpenShift, that would be Eric Rich)."

Reply:
Removed